capstone/src/main/java/com/google/univiz/CollegeScoreCardData/CollegeScoreCardInterface.java

```diff
@@ -0,0 +1,17 @@
+import java.util.List;
```

Comment:
doesn't this interface need a package declaration?

Reply:
whoops looks like I left that off! sorry about that-done!